.github/workflows/unit-test-sbomscanner.yml

```diff
@@ -0,0 +1,50 @@
+name: Unit Tests - SBOMScanner
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 21 * * *'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'pkg/sbomscanner/**'
```

Comment:
As this PR did not trigger the unit tests, I think we should also add the configuration files to the list of affected changes for the tests.